### PR TITLE
copy setup0.h to setup.h only if none already exists

### PR DESCRIPTION
**Pull request by Bigpet**
Seems like many people aren't great at reading readmes.

This might be a happy medium of not overwriting existing setup.h files but still working for the simple "open sln, hit F5" reflex that many people seem to have.

### CODE REVIEW

File: build/msw/wx_custom_build.vcxproj [hunk 1]
```diff
@@ -131,6 +131,11 @@
     <TargetName Condition="'$(Configuration)|$(Platform)'=='DLL Release|Win32'">$(wxToolkitDllNamePrefix)$(ProjectName)$(wxToolkitDllNameSuffix)</TargetName>
     <TargetName Condition="'$(Configuration)|$(Platform)'=='DLL Release|x64'">$(wxToolkitDllNamePrefix)$(ProjectName)$(wxToolkitDllNameSuffix)</TargetName>
   </PropertyGroup>
+  <ItemDefinitionGroup Condition="!Exists('..\..\include\wx\msw\setup.h')">
+    <PreBuildEvent>
+      <Command>copy ..\..\include\wx\msw\setup0.h ..\..\include\wx\msw\setup.h /Y</Command>
```
Comment (vadz) [inline build/msw/wx_custom_build.vcxproj]: Why `/Y`? Shouldn't there already be no confirmation if/as the destination file doesn't exist?

**Comment by vadz** [comment]
Thanks, this looks like a good idea, at least I don't see any problems with it and it should help the new users. I'd just like to address the comment above before merging it.

File: build/msw/wx_custom_build.vcxproj (hunk 1 above)
Comment (Bigpet) [inline build/msw/wx_custom_build.vcxproj]: Yeah, you're right, I copied that from another solution I had :P . I'll get on that

**Comment by Bigpet** [comment]
Also, I thought of another issue, what if people checkout this version, build and then update from this repository later when there's another option added?

Should there be a warning if setup0.h is newer than setup.h or something like it to help people find that potential error?

**Comment by vadz** [comment]
Normally we add `#if` checks resulting in an `#error` if some value is not defined in `setup.h` to `wx/chkconf.h`, and there is a comment in that file explaining what needs to be done, but I agree that some people probably won't read this comment neither, so it would be nice to give a warning (something like "The file wx/msw/setup0.h has been updated in the repository, update your wx/msw/setup.h, especially if you get start getting errors from wx/chkconf.h"), but I'm not sure how to do it with MSBuild (but it certainly must be possible...).

**Comment by Bigpet** [comment]
@vadz yeah I'm kind of having trouble finding a solution with msbuild that does not require C#, but I really don't know much about MSBuild either, so I might be overlooking something.

**Pull request closed without merging**